.github/workflows/ci.yml

```diff
@@ -16,12 +16,12 @@ jobs:
       fail-fast: false
       matrix:
         racket-variant: ['BC', 'CS']
-        racket-version: ['8.2']
+        racket-version: ['8.5', '8.6']
 
     steps:
       - uses: actions/checkout@v3
 
-      - uses: Bogdanp/setup-racket@b0760a463ccf5e0259d5ad925be4b36b3b6e2d08
+      - uses: Bogdanp/setup-racket@v1.9
```

Comment:
Could you pin this to the tag's git SHA? See https://exercism.org/docs/building/github/gha-best-practices#h-pin-actions-to-shas

Reply:
done!